### PR TITLE
Edits from @theandrewgraham

### PR DESCRIPTION
**Pull request by joshuaclayton**

### CODE REVIEW

File: book/application_development/geocode_browser_requests_on_the_client_side.md [hunk 1]
```diff
@@ -9,18 +9,18 @@ and [Google Geocoding][google-geocoding-api] APIs.
 ### Changes to the View
 
 In the view, we add a function which calls `getLocation()` on an instance of
-`ExampleApp.CurrentLocation` if the search field's placeholder attribute is blank.
-We pass the function a callback which sets the placeholder attribute to the
-current city and state:
+`ExampleApp.CurrentLocation` if the search field's placeholder attribute is
+blank.  We pass the function a callback which sets the placeholder attribute
```
Comment (malandrina) [inline book/application_development/geocode_browser_requests_on_the_client_side.md]: Extra space here?

File: book/application_development/geocode_browser_requests_on_the_server_side.md [hunk 2]
```diff
@@ -1,23 +1,23 @@
...
 ### Changes to the Controller and View
 
-The Geocoder gem extends the `request` object within Rails controllers with a
-new method, `#location`, which exposes information about both city and state.
-By creating a new class, `RequestGeocodingGatherer`, to handle calculating
-city and state, we're able to keep this logic out of the controller and have
-small classes, each with their own responsibility.
+The [geocoder][geocoder-github] gem extends the `request` object within Rails
+controllers with a new method, `#location`, which exposes information about
+both city and state.  By creating a new class, `RequestGeocodingGatherer`, to
```
Comment (malandrina) [inline book/application_development/geocode_browser_requests_on_the_server_side.md]: Extra space?

File: book/application_development/local_search_data.md [hunk 3]
```diff
@@ -1,24 +1,26 @@
...
 ` app/controllers/locations_controller.rb@0ce0bfa:2,8
 
-The method `.near` provided by Geocoder is flexible; at this point, we provide
-coordinates calculated by the Area gem (and rolled up into the `PostalCode`
-class). By providing coordinates, this ensures the Geocoder gem does not hit
-any external services to calculate the center of the search.
+The method `.near` provided by [geocoder][geocoder-github] is flexible; at
+this point, we provide coordinates calculated by the [area][area] gem (and
+rolled up into the `PostalCode` class). By providing coordinates, this ensures
+the [geocoder][geocoder-github] gem does not hit any external services to
```
Comment (malandrina) [inline book/application_development/local_search_data.md:21]: :+1:

File: book/application_development/geocode_browser_requests_on_the_server_side.md (hunk 2 above)
Comment (joshuaclayton) [inline book/application_development/geocode_browser_requests_on_the_server_side.md]: Fixed in aa12e0dcbd922c77cfc8b03c9c87fa701cae3d79

File: book/application_development/geocode_browser_requests_on_the_client_side.md (hunk 1 above)
Comment (joshuaclayton) [inline book/application_development/geocode_browser_requests_on_the_client_side.md]: Fixed in aa12e0d

**Comment by malandrina** [comment]
Looks good. Ready to merge.

**Pull request merged**